python/ray/air/config.py

```diff
@@ -748,7 +751,7 @@ class RunConfig:
     sync_config: Optional["SyncConfig"] = None
     checkpoint_config: Optional[CheckpointConfig] = None
     progress_reporter: Optional["ProgressReporter"] = None
-    verbose: Union[int, "Verbosity"] = 3
+    verbose: Union[int, "AirVerbosity", "Verbosity"] = 1
```

Comment:
nit: Not a huge deal but for backwards compatibility, as a user I would expect setting `RAY_AIR_NEW_OUTPUT=0` to set this back to old verbosity 3.